tests/admin.test.ts

```diff
@@ -182,3 +184,105 @@ describe('bonus points submission', () => {
     expect(getNoBonusUserResponse.user.points).toEqual(0);
   });
 });
+
+describe('event creation', () => {
+  test('successful event creation', async () => {
+    const conn = await DatabaseConnection.get();
+    const admin = UserFactory.fake({ accessType: UserAccessType.ADMIN });
+    const user = UserFactory.fake();
+
+    await new PortalState()
+      .createUsers(admin, user)
+      .write();
+
+    const event = {
+      cover: 'https://www.google.com',
+      title: 'ACM Party @ RIMAC',
+      description: 'Indoor Pool Party',
+      location: 'RIMAC',
+      committee: 'ACM',
+      start: new Date('2020-08-20T10:00:00.000Z'),
+      end: new Date('2020-08-20T12:00:00.000Z'),
+      attendanceCode: 'p4rty',
+      pointValue: 10,
+    };
+
+    const createEventRequest: CreateEventRequest = {
+      event,
+    };
+
+    const eventController = ControllerFactory.event(conn);
+    const eventResponse = await eventController.createEvent(createEventRequest, admin);
+
+    expect(eventResponse.event.cover).toEqual(event.cover);
+    expect(eventResponse.event.title).toEqual(event.title);
+    expect(eventResponse.event.location).toEqual(event.location);
+    expect(eventResponse.event.committee).toEqual(event.committee);
+    expect(eventResponse.event.title).toEqual(event.title);
+    expect(eventResponse.event.start).toEqual(event.start);
+    expect(eventResponse.event.end).toEqual(event.end);
+    expect(eventResponse.event.pointValue).toEqual(event.pointValue);
+
+    const lookupEvent = await eventController.getOneEvent({ uuid: eventResponse.event.uuid }, user);
+    expect(lookupEvent.error).toEqual(null);
+    expect(JSON.stringify(lookupEvent.event)).toEqual(JSON.stringify(eventResponse.event));
```

Comment:
Why was it necessary to stringify the events?

Reply:
I understand what Max might be trying to test here (seems like instead of comparing every single field, we stringify the whole JSON and compare it directly with the expected response), but we have a slightly different convention of asserting this - example [here](https://github.com/acmucsd/membership-portal/blob/master/tests/merchOrder.test.ts#L946). This is especially useful in the cases where you might be comparing a public schema vs. an internal model, e.g. `EventModel` vs. `PublicEvent`.

Reply:
Yeah I just wasn't really sure how to compare those two without the stringify